app/models/user.py

```diff
@@ -23,9 +25,15 @@ class Role(db.Model):
     @staticmethod
     def insert_roles():
         roles = {
-            'User': (
+            'Inactive': (
```

Comment:
OK, let's get rid of it and note that we should disable open-registration soon. Thoughts?